Examples/Algorithms/TruthTracking/ActsExamples/TruthTracking/ParticleSmearing.hpp

```diff
@@ -27,28 +28,50 @@ namespace ActsExamples {
 class ParticleSmearing final : public BareAlgorithm {
  public:
   struct Config {
+    void ReadOptions(const Options::Variables& vars) {
```

Comment:
Can this function also go into `ParticleSmearingOptions.cpp` and return a `ParticleSmearing::Config` object with the values set? That's the pattern used for other options functions, like `readRandomNumbersConfig` or `readSequencerConfig`.

Reply:
Its done